docs/source/API/core/policies/MDRangePolicy.rst

```diff
@@ -86,6 +86,11 @@ Constructors
 
     * Provide a start and end index as well as the tiling dimensions. The length of the lists must match the rank of the policy.
 
+``Notes``:
+^^^^^^^^^^
+
+* Provided start index must not be greater than the matching end index for all ranks. Otherwise, ``Kokkos::abort()`` is called.
```

Comment:
I would leave it as
```
Preconditions:
* Blah
```
That is all the user need to know.

Reply:
Applied.